chopper_generator/lib/src/generator.dart

```diff
@@ -129,21 +129,24 @@ class ChopperGenerator extends GeneratorForAnnotation<chopper.ChopperApi> {
     return Method((b) {
```

Comment:
We should avoid using one-letter variable and parameter names for readability's sake. It's not the scope of this PR, so I'll let it pass, but we should really do it.

Reply:
Yeah, totally agree. Let's push that after this PR. Also, we should decide how to release new versions of Chopper.